### DISCUSSION TITLE
Cruiser navigation offtrack rotating guiding arrow.

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#259] [Q&A] [answered]
Navigation > No autorecalculation.

Start the navigation from an "off track" position.

The Cruiser app in the top left navigation panel than shows you the off track rotating **Guiding** Arrow.
The Guiding arrow does not "point" toward the Start position(Via Point 0) but instead "points" toward the Via Point 1.
The Guiding arrow should not skip the start position instead should guide you toward the Start (Via point 0) position.
At the meeting start position Cruiser than should as usual continue with the standard "on track" navigation.

Video:  https://youtu.be/Rr1sBcxtWa8

**Comment by devemux86** [comment]
@0709wiwiwi thanks for the detailed report!

> Start the navigation from an "off track" position.
> The Guiding arrow should not skip the start position instead should guide you toward the Start (Via point 0) position.

It will work like this in the next Cruiser update.

So it will be consistent with the rerouting dialog, also offering the start and all waypoints.

**Comment by devemux86** [comment, marked as answer]
Implemented in [Cruiser 3.0.27](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
